### ISSUE TITLE
Whole integration not working anymore

### ISSUE THREAD

**Issue by BeyondPixels** [albertogeniola/meross-homeassistant#211]
**Describe the bug**
For several days now, all my Meross sensors has stopped working.
The page on HA website doesn't exist anymore:
https://www.home-assistant.io/components/meross_cloud

I've tried to uninstall and reinstall it all (HACS then HA integrations) but got the following error message:
"Only a single configuration of meross is allowed"

**Your environment**
HomeAssistant version: 2021.2.3
Hassio Version (if applicable): -- PLEASE SPECIFY --
Hardware environment: Raspberry Pi 3

**Comment by BeyondPixels** [comment]
I'd like to add some information.

So, I retried to install it all.

Here is what happens (same as before):
![Meross-problem](https://user-images.githubusercontent.com/53296978/111965101-a54f7280-8af5-11eb-8de9-880e7fd18bfe.gif)

The wait before the error message is actually longer than that, I skipped more than half of it.

After that, I can see the integration despite the error but there isn't any device nor entities configured with it.


Any help would be greatly appreciated,
Thanks in advance.
